extensions/wikia/AdEngine/js/video/googleIma.js

```diff
@@ -117,13 +122,18 @@ define('ext.wikia.adEngine.video.googleIma', [
 						self.adDisplayContainer.initialize();
 						self.adsManager.init(width, height, google.ima.ViewMode.NORMAL);
 						self.adsManager.start();
+						console.log('***** VIDEO PLAY STARTED');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)